packages/origin-ui-core/src/utils/exchange/types.ts

```diff
@@ -159,6 +159,7 @@ export type Bundle = {
     isCancelled: boolean;
     items: BundleItem[];
     volume: BigNumber;
+    own: boolean;
```

Comment:
What does this variable mean?

Reply:
It means that this bundle is owned by the logged in user